platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -84,6 +85,15 @@ COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
 COPY ["files/configdb-load.sh", "/usr/bin/"]
 COPY ["files/arp_update", "/usr/bin"]
 
+COPY ["interfaces.j2", "/usr/share/sonic/templates/"]
+COPY ["deployment_id_asn_map.yml", "/etc/sonic/"]
+COPY ["teamd.sh", "/usr/bin/"]
+COPY ["teamd.j2", "/usr/share/sonic/templates/"]
+COPY ["bgpcfgd", "/usr/bin/"]
+COPY ["sonic_version.yml", "/etc/sonic/sonic_version.yml"]
+COPY ["*.j2", "/usr/share/sonic/templates/"]
+COPY ["minigraph.xml", "/etc/sonic/"]
+
```

Comment:
Copied files. Keep them in sync with the latest by adding it to the makefile/rules.mk